app/containers/Game/Game.js

```diff
@@ -44,7 +43,6 @@ export const Game = ({roomId, roomConnected, state}) => {
       <GamePlayers started={started} remotePlayers={remotePlayers} />
       {!started &&
         <React.Fragment>
```

Comment:
this `React.Fragment` component is a way to avoid needing to place arbitrary `divs` when there are more than one child component. Since you removed `GamePlayersController`, theres only one child component now - `GameConfig`. As such, you don't need the Fragment wrapping it anymore 👍